android/app/src/main/java/com/chorestory/app/ParentQuestDetailsActivity.java

```diff
@@ -122,14 +103,14 @@ public void onClick(View v) {
                 DatePickerDialog.OnDateSetListener listener = new DatePickerDialog.OnDateSetListener() {
                     @Override
                     public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
-                        month++; // month is represented by an integer from 0 to 11
                         mDay = dayOfMonth;
                         mMonth = month;
```

Comment:
month is represented by an integer from 0 to 11, since you incremented by 1 in getDateText()
should we do this for all other activities/fragments to be consistent

and what is the reasoning for this change

Reply:
changed because when I fetch the time from the server it's 0-11, so the only time we want 1-12 is when we display it on the screen. the other file doesn't fetch anything from the server that's why it's easier to just immediately add 1. I could change the file if you really want it to be the same but can we just do this later

Reply:
also if i change the other file now I'll get merge conflicts so can we just change in different pr plz

Reply:
Make sure that someone will do this!